modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeAutoBatchMethodCheck.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.SourceUtil;
+import com.liferay.source.formatter.parser.JavaClass;
+import com.liferay.source.formatter.parser.JavaClassParser;
+import com.liferay.source.formatter.parser.JavaMethod;
+import com.liferay.source.formatter.parser.JavaTerm;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Regisson Aguiar
+ */
+public class UpgradeAutoBatchMethodCheck extends BaseUpgradeCheck {
+
+	@Override
+	protected String format(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
+
+		if (javaClass == null) {
+			return content;
+		}
+
+		for (JavaTerm javaTerm : javaClass.getChildJavaTerms()) {
+			if (!javaTerm.isJavaMethod()) {
+				continue;
+			}
+
+			JavaMethod javaMethod = (JavaMethod)javaTerm;
+
+			String javaMethodContent = javaMethod.getContent();
+
+			content = _formatContent(
+				javaClass.getExtendedClassNames(), content, fileName,
+				javaMethodContent);
+		}
+
+		return content;
+	}
+
+	private String _formatContent(
+		List<String> classNames, String fileContent, String fileName,
```

Comment:
Hi @regiCesar21 , there is still `content` and `fileContent`, if they have the same meaning then they should have the same name.